### PR TITLE
Add API Designer Feature 

### PR DESCRIPTION
**Pull request by DimalChandrasiri**

### CODE REVIEW

File: composer/packages/api-designer/package.json [hunk 1]
```diff
@@ -4,11 +4,33 @@
...
+  "dependencies": {
+    "@types/swagger-parser": "^4.0.2",
+    "@types/uniqid": "^4.1.2",
+    "js-yaml": "^3.12.0",
+    "semantic-ui-css": "^2.4.1",
+    "semantic-ui-react": "^0.83.0",
+    "swagger-parser": "^5.0.5",
+    "swagger-schema-official": "^2.0.0-bab6bed",
+    "uniqid": "^5.0.3",
+    "url-loader": "^1.1.1"
```
Comment (kaviththiranga) [inline composer/packages/api-designer/package.json]: do we need the url-loader here?

File: composer/packages/api-designer/src/api-designer.ts [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ *
+ */
+
+import { createElement } from "react";
+import { render } from "react-dom";
+import OpenApiVisualizer from "./api-designer/index";
```
Comment (kaviththiranga) [inline composer/packages/api-designer/src/api-designer.ts]: /index part is redundant

File: tool-plugins/vscode/package.json [hunk 3]
```diff
@@ -188,8 +188,14 @@
                 "category": "Ballerina"
             },
             {
-                "command": "ballerina.showTraces",
```
Comment (kaviththiranga) [inline tool-plugins/vscode/package.json]: show traces command is removed. plz fix and @pahans  plz verify

File: composer/packages/distribution/webpack.config.js [hunk 4]
```diff
@@ -11,10 +11,10 @@ const ExtractVSCodeThemeCSS = new ExtractTextPlugin({
 
 module.exports = {
     output: {
-        path: path.resolve(__dirname, 'build'),
-        filename: 'composer.js',
-        library: 'ballerinaComposer',
-        libraryTarget: 'umd'
+      path: path.resolve(__dirname, 'build'),
+      filename: '[name].js',
```
Comment (kaviththiranga) [inline composer/packages/distribution/webpack.config.js]: for this to work you should have multiple entry points too. plz verify

File: composer/packages/api-designer/src/api-designer.ts (hunk 2 above)
Comment (DimalChandrasiri) [reply composer/packages/api-designer/src/api-designer.ts]: Fixed with commit 494035b

**Pull request merged**